unraid_mcp/tools/keys.py

```
@@ -0,0 +1,146 @@
+                keys = data.get("apiKeys", [])
+                return {"keys": list(keys) if isinstance(keys, list) else []}
+
+            if action == "get":
+                if not key_id:
+                    raise ToolError("key_id is required for 'get' action")
+                data = await make_graphql_request(QUERIES["get"], {"id": key_id})
+                return dict(data.get("apiKey", {}))
+
+            if action == "create":
+                if not name:
+                    raise ToolError("name is required for 'create' action")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)